system_metrics_collector/test/system_metrics_collector/test_linux_process_cpu_measurement_node.cpp

```diff
@@ -252,122 +129,187 @@ class LinuxProcessCpuMeasurementTestFixture : public ::testing::Test
 
   void TearDown() override
   {
-    test_node_->shutdown();
-    EXPECT_FALSE(test_node_->IsStarted());
-    EXPECT_EQ(State::PRIMARY_STATE_FINALIZED, test_node_->get_current_state().id());
+    test_measure_linux_process_cpu_->shutdown();
+    EXPECT_FALSE(test_measure_linux_process_cpu_->IsStarted());
+    EXPECT_EQ(
+      State::PRIMARY_STATE_FINALIZED,
+      test_measure_linux_process_cpu_->get_current_state().id());
 
-    test_node_.reset();
+    test_measure_linux_process_cpu_.reset();
     rclcpp::shutdown();
   }
 
 protected:
-  std::shared_ptr<MockLinuxProcessCpuMeasurementNode> test_node_;
+  std::shared_ptr<MockLinuxProcessCpuMeasurementNode> test_measure_linux_process_cpu_;
 };
 
 TEST_F(LinuxProcessCpuMeasurementTestFixture, TestGetMetricName) {
   const int pid = system_metrics_collector::GetPid();
-  ASSERT_EQ(std::to_string(pid) + "_cpu_percent_used", test_node_->GetMetricName());
+  ASSERT_EQ(
+    std::to_string(
+      pid) + "_cpu_percent_used", test_measure_linux_process_cpu_->GetMetricName());
 }
 
 TEST_F(LinuxProcessCpuMeasurementTestFixture, TestManualMeasurement)
 {
   // first measurement caches
-  double cpu_active_percentage = test_node_->PeriodicMeasurement();
+  double cpu_active_percentage = test_measure_linux_process_cpu_->PeriodicMeasurement();
```

Comment:
`const auto` unless you're performing implicit casting. In which case, please explicitly cast.

Reply:
Can use auto but not const (see line 160).